test/integration.test.ts

```diff
@@ -150,4 +154,44 @@ describe('Starting cardano-wallet (and its node)', () => {
       }),
     longTestTimeoutMs
   );
+
+  describe('Child process logging support', () => {
+    const tmpDir = './test/.tmp';
```

Comment:
It would be better to use the `tmp.dir()` function for this, rather than writing files in the source tree.